### PR TITLE
Iterate on notebooks and Add configuration files

### PR DESCRIPTION
**Pull request by IvanPombo**
In this PR, I iterate over the datasets, clean everything and test them to see if they are running.

There is still some features missing on Gromacs, but there is still time on Monday to finish it.
Moreover, the design exploration will pass to the API today so the notebook will also be simplified.

Upfront, I'm sorry for such a big update, but I inserted the configuration files in this folder.

### CODE REVIEW

File: demos/tutorials/5_molecular_dynamics.ipynb
Comment (hpenedones) [inline demos/tutorials/5_molecular_dynamics.ipynb:1]: This notebooks contains an API key from someone (which now we should invalidate).

File: demos/tutorials/4_design_exploration.ipynb
Comment (hpenedones) [inline demos/tutorials/4_design_exploration.ipynb:1]: This was is pre-rendering quite a few error messages:
image

File: demos/tutorials/5_molecular_dynamics.ipynb
Comment (IvanPombo) [reply demos/tutorials/5_molecular_dynamics.ipynb:1]: Hawk eyes! Just removed it.

File: demos/tutorials/4_design_exploration.ipynb
Comment (IvanPombo) [reply demos/tutorials/4_design_exploration.ipynb:1]: That one is still being updated by @luispcunha.

**Review by luispcunha — APPROVED** [review]

**Pull request merged**